build/Dockerfile-aws-dev

```diff
@@ -2,10 +2,8 @@ FROM frolvlad/alpine-glibc
 RUN apk add --no-cache ca-certificates postgresql-client
 ADD rudder-server /
 ADD rudder-server-with-race /
-RUN mkdir /app
-ADD . /app
```

Comment:
We also need config.toml file in the image at path /app/config/config.toml

Reply:
Added